bin/genesis

```diff
@@ -4069,12 +4191,17 @@ sub active_certificates {
 sub target_vault {
 	my ($target) = @_;
 	if ($target) {
-		system(qw(safe target), $target) == 0 or exit 1;
+		Genesis::ShellExec::interact({passfail => 1}, 'safe target "$1"', $target) or exit 1;
 	} else {
-		system(qw(safe target -i)) == 0 or exit 1;
+		Genesis::ShellExec::interact({passfail => 1}, 'safe target -i') or exit 1;
```

Comment:
Ours is not to question why.

Ours is but to do_or_die()

(with interact => 1)

Reply:
Migrate the if ($target) call into the arg list of the interact.

The target is either set, or it's `-i`